docs/source/features/dataset_management.rst

```diff
@@ -62,19 +57,18 @@ There are plenty of benefits of uploading local datasets to TensorBay.
 Two types of datasets can be read from TensorBay:
 
 - Datasets uploaded by yourself as mentioned in :ref:`features/dataset_management:Upload Dataset`.
-- Datasets uplaoded by the community (i.e. the `open datasets`_).
+- Datasets uploaded by the community (i.e. the `Open Datasets`_ platform).
+
+Take the :ref:`Read of BSTLD <examples/bstld:Read Dataset>` as an example.
 
 .. note::
 
    Before reading a dataset uploaded by the community, fork_ it first.
 
 .. note::
 
-   Visit our `Graviti AI Service(GAS)`_ platform to check the dataset details,
+   Visit our `Open Datasets`_ platform to check the dataset details,
```

Comment:
Visit `my datasets(or team datasets)`_ panel of `TensorBay`_ platform to check all datasets that can be read.
.. _my datasets(or team datasets): https://gas.graviti.cn/tensorbay/dataset-list